server_versions.json

```diff
@@ -81,50 +77,18 @@
         "version": "1.19.2",
         "link": "https://api.papermc.io/v2/projects/paper/versions/1.19.2/builds/307/downloads/paper-1.19.2-307.jar"
       },
-      {
-        "version": "1.19.1",
-        "link": "https://api.papermc.io/v2/projects/paper/versions/1.19.1/builds/111/downloads/paper-1.19.1-111.jar"
-      },
-      {
-        "version": "1.19",
```

Comment:
Why did you remove this? I would leave it there

Reply:
The most used versions of the 1.19-Group are 1.19.2 and 1.19.4, the 1.19 is outdated and 1.19.2 fixing some bugs of 1.19.1 and lower